### ISSUE TITLE
`can't execute OpAssign in a macro` error with Crystal 0.20

### ISSUE THREAD

**Issue by ruivieira** [drujensen/kemalyst-model#15]
OS: macOS 10.12.1
Crystal:  0.20.0 (2016-11-22)
kemalyst-model: latest (shards)

```
require "./db.cr"
^

in src/db.cr:6: expanding macro

  sql_mapping({
  ^~~~~~~~~~~

in lib/kemalyst-model/src/kemalyst-model.cr:57: can't execute OpAssign in a macro

        {% i += 1 %}
```

**Comment by drujensen** [comment]
Released v0.2.3 that supports Crystal 0.20.0

**Comment by ruivieira** [comment]
Working fine from my side, thanks!
